app/views/external_users/claims/interim_fee/_fields.html.haml

```diff
@@ -85,6 +85,6 @@
 
     %hr
 
-    .form-group
+    #disbursements.form-group
```

Comment:
Should this be `interim_fees`?  

Or are disbursements sometimes known as different things?

Reply:
There is that strange combination form on interim fees.. This ID is used as a hook to scope the disbursement functionality